src/app/app.module.ts

```diff
@@ -218,6 +223,10 @@ import { CountdownTimerComponent } from "./countdown-timer/countdown-timer.compo
     PopoverModule.forRoot(),
     RatingModule.forRoot(),
     CollapseModule.forRoot(),
+    ThemeModule.forRoot({
+      themes: [lightTheme, darkTheme],
+      active: localStorage.getItem('theme') || 'dark'
```

Comment:
Suggestion for defaulting to the system theme, if you wanted. Should also have input from @chafreaky since it was his recommendation:

```ts
active: localStorage.getItem('theme') || (
    window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light'
)
```

Reply:
Hmmm, not a bad idea, would be happy to add it, will await @chafreaky reply